cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -308,6 +310,7 @@ pub type TrustedReserves = (
 	AssetsFrom<SystemAssetHubLocation>,
 	NativeAssetFrom<SystemAssetHubLocation>,
 	AssetPrefixFrom<CustomizableAssetFromSystemAssetHub, SystemAssetHubLocation>,
+	AssetPrefixFrom<BridgedAssetPrefix, SystemAssetHubLocation>,
```

Comment:
@alistair-singh  Actually `CustomizableAssetFromSystemAssetHub` is a [configurable storage](https://github.com/Snowfork/polkadot-sdk/blob/df4bad7394abb8104c31e91165b45d45d7556cc0/cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs#L286-L291) which can be configured with a `System.set_storage` as any location like Ethereum.

But I'm also fine with the change here.